.travis.yml

```diff
@@ -4,7 +4,7 @@ env:
     - SHARED=OFF
     - BUILD_DEPENDENCY_FROM_SOURCE=OFF
     - BUILD_TENSORFLOW_OPS=ON
-    - NPROC=2
+    - NPROC=1
```

Comment:
Sure, I don't see why not. But NPROC=1 is going too slowly for macOS anyway, so it's been changed back to 2 since you reviewed it. One of the macOS builds fails, so further changes may be necessary.

Reply:
Ok, let's wait for the other macos changes